libcodechecker/server/database/database_status.py

```diff
@@ -0,0 +1,24 @@
+
+# -------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -------------------------------------------------------------------------
+
+"""
+Map between database statuses and the corresponding messages.
+"""
+from shared.ttypes import DBStatus
+
+db_status_msg = {
+    DBStatus.OK: "Database is up to date.",
+    DBStatus.MISSING: "Missing database.",
+    DBStatus.FAILED_TO_CONNECT: "Connection failed.",
+    DBStatus.SCHEMA_MISMATCH_OK:
+        "Database schema mismatch: migration is available.",
```

Comment:
We use `migration` here, and in the code, but the command is called **`upgrade`** schema. Perhaps it would be more user-friendly if we said: `Database schema mismatch! Possible to upgrade.`